src/pages/components/pagination/usage.mdx

```diff
@@ -54,55 +59,377 @@ control for navigating to the next or previous page.
+- When there is too much data to display on one page or within one view of a
+  component
+- To make large amounts of data more accessible to consume by users
+- To optimize on-page real estate
+- To give users more control over how they view large amounts of information
+
+### When not to use
+
+- Do not use it to display linear journeys, for example, in a form progression.
+  Instead, use the progress bar or button components to navigate forward and
+  backward.
+- Do not use pagination superfluously, and aim to use it to improve usability or
+  performance.
+
+### Variants
+
+| Variant                           | Usage                                                                                                                           |
+| --------------------------------- | ------------------------------------------------------------------------------------------------------------------------------- | --- |
+| [Pagination](#pagination)         | The pagination variant is typically connected at the bottom of the data table component to help paginate large amounts of data. |
+| [Pagination nav](#pagination-nav) | The pagination nav variant is mainly used in on-page situations to help paginate either a whole page or sections of a page.     |     |
+
+## Formatting
+
+### Anatomy of pagination
+
+<Row>
+<Column colLg={12}>
+
+![Anatomy of the pagination variant](images/pagination-usage-anatomy.png)
+
+</Column>
+</Row>
+
+<Caption>Anatomy of the pagination variant</Caption>
+
+1. **Items per page:** Shows the current number of items that appear per page.
+2. **Range of items:** Shows the current range of items and lists the total
+   amount of items.
+3. **Current page:** Shows the current page the user is on and lists the total
+   amount of pages. 4.**Previous and next buttons:** Buttons that take you to
+   the previous or next page.
+4. **Container:** The container bar of the whole pagination component.
+
+### Anatomy of pagination nav
+
+<Row>
+<Column colLg={12}>
+
+![Anatomy of the pagination nav variant](images/pagination-usage-anatomy-nav.png)
+
+</Column>
+</Row>
+
+<Caption>Anatomy of the pagination nav variant</Caption>
+
+1. **Selected page button:** The current page the user is viewing.
+2. **Unselected page button:** A page the user could select and view.
+3. **Overflow button:** Contains available pages that can be navigated to but
+   cannot be shown upfront in the component because of the current viewport.
+4. **Previous and next buttons:** Buttons that take you to the previous or next
+   page.
+5. **Container:** The container of the whole pagination component.
+
+### Sizing
+
+The pagination component is available in three sizes—**large**, **medium**, and
+**small**.
+
+<Row>
+<Column colLg={12}>
+
+![Sizes of the pagination component](images/pagination-usage-sizes.png)
+
+</Column>
+</Row>
+
+<Caption>Sizes of the pagination component</Caption>
+
+We recommend using the same height pagination as you do for the data table
```

Comment:
I feel like this needs an h4 to help with scannability. Maybe something like: Data table size pairings

Reply:
Totally!